tests/fw/comm/wscript

```diff
@@ -2,6 +2,11 @@ from waftools.pebble_test import clar
 import waftools.ragel
 
 def build(bld):
+    # These tests don't compile because the bluetooth library needed was removed. The 'COMM_TESTS_DISABLED'
+    # flag has been added to temporarily disabled them.
+    if 'COMM_TESTS_DISABLED' in bld.env.DEFINES:
```

Comment:
This feels like the wrong place to put this flag, since you're not. actually using it in C code. It should be just part of the waf env itself I think?

Reply:
Agreed. This specific flag has been replaced by the test exclusion list, but I did add a new flag for skipping the python tool tests. That flag has been added directly to env.